gfam/tasks/find_unassigned/task.py

```diff
@@ -28,20 +28,24 @@ def __init__(self,
             self.log = logger
         self.sequence_id_regexp = sequence_id_regexp
 
+        # these are moved here from get_unassigned
+        self.regions = []
+        self.maximum = None
```

Comment:
It was already in the previous version but I think I don't understand the role of this variable, can you clarify?

Reply:
self.regions was originally built in `get_unassigned`, it collects unassigned regions

self.maximum as well (this is the maxmin length of the unassigned fragments, later used as a minimum for the fasta regions)